docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,123 @@
+|  [Control Flow, Congestion Avoidance/Control][5] | Yes               | No                       |  Yes, and user controlled                                          
+
+## 2. The 5 Transport Guarantees
+
+There are 5 main different ways you can transfer data:
+
+| Transport Guarantees         | Packet Drop [(1)][1]  | Packet Duplication [(2)][2] | Packet Order | Packet Delivery |
+| :-------------:              | :-------------: | :-------------: | :-------------: | :-------------:
+|   **Unreliable Unordered**   |       Any       |      Yes        |     No          |   No
+|   **Unreliable Sequenced**   |    Any + old    |      No         |     Sequenced   |   No
+|   **Reliable Unordered**     |       No        |      No         |     No          |   Yes
+|   **Reliable Ordered**       |       No        |      No         |     Ordered     |   Yes
+|   **Reliable Sequenced**     |    Only old     |      No         |     Sequenced   |   Only newest
+
+Unreliability gives great uncertainty with a lot of freedom, while reliability gives great certainty with costs in speed and freedom.
+That is why protocols such as QUIC, RUDP, SCTP, QUIC, netcode, laminar are built on UDP instead of TCP. 
+UDP gives the end-user more control over the transmission than TCP can do. 
+
+## 3. Issues with TCP 
+
+Now the golden question: "Why choose so much uncertainty with UDP when TCP is so reliable and safe?". 
+To answer that question we will have to delve a little deeper into some issues with TCP. 
+
+### Head-of-line Blocking
+
+One of the biggest issues with TCP is that of Head-of-line blocking. 
+It is a convenient feature because it ensures that all packages are sent and arrive in [order][guarantees]. 
+However, in cases of high throughput (multiplayer game networking) and big load in a short time (web page load), this can be catastrophic to your application performance.
+
+The issue is demonstrated in the following animation:
+
+![Head of line blocking][animation] 
+
+This animation shows that if a certain packet drops in transmission, all packets have to wait at the transport layer until it is resent by the other end.
+If the dropped packet is resent and arrived then all packets are freed from the transport layer. 
+
+Let's look at two areas where this head-of-line blocking issue is a huge deal.
+
+**Multiplayer Game Networking**
+
+Multiplayer action games are based on a constant stream of packets sent at a speed ranging from 10 to 30 packets per second.
+For the most part, the data in these packages are so time-sensitive that only the most recent data is useful.
+You can think of the input and position of the player, the orientation and speed, and the state of the physical objects in the world.
+If a single packet drops out we can not afford to queue up 10-30 packets a second until the lost packet is retransmitted. 
+This could cause annoying lag behavior and a bad user experience. 
```

Comment:
While QUIC might work well for game networking and this is the reason you are interested in it, I think we should mention web networking first, as this was the reason it was designed in the first place. Because it is well designed, it also benefits the types of things for game networking, but this seems almost accidental.

Reply:
Good idea, I had not a specific reason for it